tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/internal/liferay-upgrade-plan.markdown

```diff
@@ -11,458 +11,458 @@ Here are the steps for upgrading Liferay and your custom plugins to @product-ver
 ## Upgrade Steps
 
 <ol id="root">
-	<li icon="" requirement="required">
+	<li imagePath="" requirement="required">
```

Comment:
Hey Terry,  we can leave this out until we actually have icons.  Can you just remove the attibutes for right now?

Reply:
Also I've sent our changes to Jim over here:  https://github.com/jhinkey/liferay-docs/pull/444